### DISCUSSION TITLE
getting workflow faulted on using timer

### DISCUSSION THREAD

**Discussion by vabs212** [elsa-workflows/elsa-core#1457] [Design & tech]
Hi Getting the below error while implementing timer
![image](https://user-images.githubusercontent.com/66916559/131482416-d120de2e-2120-4cc5-9e1e-134f957ef1eb.png)

Please help.
O implementing timer can not know the exact details >Please tell how to debug this issue or any help

**Comment by sfmskywalker** [comment]
Can you share your workflow so I can take a look? I suspect you used an unsupported value for the Interval expression.

I admit that the error message isn't very helpful. but an update was pushed today that should include the full exception hierarchy.

**Reply by vabs212** [reply]
I used 00:00;00:70 as the expression in timer>which gave the above result. it was neither able to take integer value nor string. I checked the Code and see that you are using Noda time expression and in it it is setting the value in Duration timeout property but it is not able to set the above expression in Duration timeout property.

**Reply by sfmskywalker** [reply]
I see two issues with the expression when you try to get it parses with Nodatime's `Duration` primitive (which is what Elsa uses):

1. There's a semicolon which is unsupported.
2. The value `70` overflows the maximum of 60 seconds in a minute.

The following expression will work: 

`00:00:01:10`

The pattern is:

`day : hour : minute : second`
